libraries/bezier-rs/src/bezier/lookup.rs

```diff
@@ -4,6 +4,21 @@ use crate::utils::{TValue, TValueType};
 /// Functionality relating to looking up properties of the `Bezier` or points along the `Bezier`.
 impl Bezier {
 	/// Convert a euclidean distance ratio along the `Bezier` curve to a parametric `t`-value.
+	/// Converts a Euclidean (x, y) coordinate to a parametric representation along a curve segment.
```

Comment:
This is misinformation. Please see the comment above for what the function actually does.